src/transaction/log_manager.c

```diff
@@ -10678,7 +10679,7 @@ cdc_log_extract (THREAD_ENTRY * thread_p, LOG_LSA * process_lsa, CDC_LOGINFO_ENT
 	/*supplemental log info types : time, tran_user, undo image */
 	LOG_REC_SUPPLEMENT *supplement;
 	int supplement_length;
-	SUPPLEMENT_REC_TYPE rec_type;
+	SUPPLEMENT_REC_TYPE rec_type = LOG_SUPPLEMENT_LARGER_REC_TYPE;
```

Comment:
A question:
Why does only `rec_type` has to be initialized, not `supplement` and the other? I don't mean for you to fix it. I actually don't think rec_type has to be initialized. I'd just like to know they Valgrind complains only about this.

Reply:
Yes, Valgrind checks the initialization of variables which are used for conditional jump (switch (rec_type) case)

Reply:
thank you for the information :)